src/docs/release-notes/R29/FullNotes.mdx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
```

Comment:
There should be a mention of the new Figma Component Sticker Sheet. I made a new sticker sheet and updated everything to Material Design 3.

Reply:
Here's a link you may want to include: https://www.figma.com/community/file/1293308181354933598/brightlayer-ui-tokens-components-in-material-design-3